jenkins-integration/jenkins-jobs/common/scripts/jenkins/pipeline.groovy

```diff
@@ -284,6 +284,17 @@ def step080_customize_java_args(script_dir, server_java_args, server_era) {
     }
 }
 
+def step081_customize_jruby_jar(script_dir, jruby_jar, server_era) {
+    if ((jruby_jar == null) || (jruby_jar == "")) {
+        echo "Skipping jruby jar configuration because none specified in job"
```

Comment:
Yeah I think that'd be fine. I was initially thinking that it should be managed explicitly by the groovy job definitions, but yeah having it clear the variable between jobs is probably what a user would expect/want